index.html

```diff
@@ -0,0 +1,607 @@
+              type="text"
+              class="form-control"
+              id="skillName"
+              placeholder="Web Development"
+            />
+          </div>
+          <div class="form-group col-md-4">
+            <label for="level">Level</label>
+            <input
+              type="text"
+              class="form-control"
+              id="level"
+              placeholder="Master"
+            />
+          </div>
+          <div class="form-group col-md-4">
+            <label for="keywords">Keywords</label>
+            <input
+              type="text"
+              class="form-control"
+              id="keywords"
+              placeholder="'HTML','CSS','Javascript'"
+            />
+          </div>
+        </div>
+        <!-- Languages -->
+        <hr />
+        <h4>Languages</h4>
+        <div class="form-row">
+          <div class="form-group col-md-6">
+            <label for="language">Language</label>
+            <input
+              type="text"
+              class="form-control"
+              id="language"
+              placeholder="English"
+            />
+          </div>
+          <div class="form-group col-md-6">
+            <label for="fluency">Fluency</label>
+            <input
+              type="text"
+              class="form-control"
+              id="fluency"
+              placeholder="Native Speaker"
+            />
+          </div>
+        </div>
+        <!-- Interests -->
+        <hr />
+        <h4>Interests</h4>
+        <div class="form-row">
+          <div class="form-group col-md-6">
+            <label for="interestName">Name</label>
+            <input
+              type="text"
+              class="form-control"
+              id="interestName"
+              placeholder="Wildlife"
+            />
+          </div>
+          <div class="form-group col-md-6">
+            <label for="interestKeyword">keywords</label>
+            <input
+              type="text"
+              class="form-control"
+              id="interestKeyword"
+              placeholder="'Ferrets','Unicorns'"
+            />
+          </div>
+        </div>
+        <!-- References -->
+        <hr />
+        <h4>References</h4>
+        <div class="form-row">
+          <div class="form-group col-md-6">
+            <label for="refName">Name</label>
+            <input
+              type="text"
+              class="form-control"
+              id="refName"
+              placeholder="Jane Doe"
+            />
+          </div>
+          <div class="form-group col-md-6">
+            <label for="reference">Reference</label>
+            <input
+              type="text"
+              class="form-control"
+              id="reference"
+              placeholder="Reference..."
+            />
+          </div>
+        </div>
+        <!-- Submit -->
+        <button type="submit" class="btn btn-primary mb-3">Create JSON</button>
+      </form>
+    </div>
+
+    <script>
```

Comment:
I think it's awesome that you created a form in your front end. However, it might have been more productive to find a way to introduce the information from the JSON object in a way that wasn't hard coded. For example, you could have created a for loop that created these elements with the JSON info